CHANGES.txt

```diff
@@ -6,6 +6,12 @@ Unreleased
 ==========
 
 
+2023/04/18 0.4.1
```

Comment:
As 0.4.1 is already referenced, should it be updated in `setup.py` as well?

Reply:
Thanks. I've also updated `setup.py`, separating the release commit 114cb6d24c5 from the code change commit 29051b1cdc2.